test_4062.py

```diff
@@ -0,0 +1,29 @@
+import setupext
```

Comment:
Can you please put this in the testing directory with the rest of the tests?  We use nose every where else instead of unit test.